src/crystalla/matrix.cr

```diff
@@ -104,6 +121,14 @@ module Crystalla
       Matrix.new(added, number_of_rows, number_of_cols)
     end
 
+    def -
+      m = self.clone
+      m.values.size.times do |i|
```

Comment:
Fixed for `-` on 6591eac
How would you do it for `+`? I was considering a `zip` and then a `map`, but I didn't want to create the intermediate array. Or is `zip` lazy?